src/cards/summon_generic.ts

```diff
@@ -143,7 +143,7 @@ export default function makeSpellForUnitId(unitId: string, asMiniboss: boolean,
 🗡️ ${unitSource.unitProps.damage} ${i18n(['damage'])}` : ''}${!!unitSource.unitProps.attackRange ? `
 🎯 ${unitSource.unitProps.attackRange} ${i18n(['attack range'])}` : ''}
 ❤️ ${healthMax} ${i18n(['health capacity'])}
-${manaMax ? `🔵 ${manaMax} + ${unitSource.unitProps.manaPerTurn} ${i18n('Mana')} ${i18n('per turn')}` : ''}`;
+    ${manaMax ? `🔵 ${manaMax} + ${unitSource.unitProps.manaPerTurn} ${i18n('Mana')} ${i18n('per turn')}` : ''}`;
```

Comment:
Is this whitespace intentional? In a template literal string it will preserve the whitespace so this line will be out of alignment with the others.

Reply:
This whitespace change is not intentional.